app/models/application.py

```diff
@@ -44,25 +40,47 @@ def from_dict(cls, d: dict):
             }
         )
 
-    def create_sections(self, application):
+    @staticmethod
+    def create_blank_sections(fund_id, round_id, language):
+        sections = []
```

Comment:
Could this be removed, as seems to be created further down (in line 56?)

Reply:
removing this 👍🏾